src/current/_includes/v23.1/ui/transactions-filter.md

```diff
@@ -6,14 +6,39 @@
   {% assign page_prefix = "ui-" %}
 {% endif %}
 
+## Transaction Fingerprints results
+
+The Transaction Fingerprints returned are determined by the selected **Search Criteria**:
+
+1. By default, the **Top** `100` Transaction Fingerprints **By** `% of All Runtime` are returned.
```

Comment:
Same feedback as the statements fingerprints section re: proper noun and active voice